rocketpy/AeroSurface.py

```diff
@@ -159,6 +167,7 @@ def __init__(
         self._rocketRadius = rocketRadius
         self._baseRadius = baseRadius
         self._length = length
+        self.bluffiness = bluffiness
```

Comment:
I'm not an expert in English skills, but my text suggester is telling me that "bluffness" (without the "i") would sound better

Reply:
You are right. 
Already corrected.